docs/deployment/managed-scanning.md

```diff
@@ -174,4 +177,9 @@ To **view** the repositories you have granted access to:
 
 ### Scan logs and statistics
 
-Scan statistics, such as how many of your repositories are being scanned, the scan success rate, and so on, can be provided once a week upon request. Contact your Semgrep account manager to request scan statistics.
+You can view logs for your most recent scan by clicking **Projects > the project's latest scan time** under **Scan status**.
```

Comment:
Scan logs _should_ be available shortly after the scan is finished running, but it is possible that there isn't an associated log if something goes wrong. Not sure if that's worth mentioning in the docs though.

Reply:
Maybe this would be worth a troubleshooting mention at the end?

Reply:
Added a callout: 

![image](https://github.com/user-attachments/assets/dd7b6151-8c7f-490f-ab3e-a51e8690945d)